### PR TITLE
homework.lambda.part1.exercise1 (yakushenko)

### PR DESCRIPTION
**Pull request by Dgafar**

### CODE REVIEW

**Review by vasily-tolkachev — APPROVED** [review]

File: src/test/java/lambda/part1/exercise/Exercise1.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    @Test
+    void sortPersonsByAgeUsingArraysSortLocalComparator() {
+        Person[] persons = getPersons();
+
+        class ComparatorByAge implements Comparator<Person> {
+
+            @Override
+            public int compare(Person o1, Person o2) {
+                return o1.getAge() - o2.getAge();
```
Comment (Ilya21) [inline src/test/java/lambda/part1/exercise/Exercise1.java:27]: I think it's better to use Integer.Compare because of  boundary values

File: src/test/java/lambda/part1/exercise/Exercise1.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    @Test
+    void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
+                if (o1.getLastName().equals(o2.getLastName())) {
```
Comment (Ilya21) [inline src/test/java/lambda/part1/exercise/Exercise1.java:65]: You can extract `o1.getLastName().equals(o2.getLastName()` into local value. It will help to avoid duplicate code

**Review by TarasKobzar — APPROVED** [review]
